include/cell/compute/map.hpp

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "cell/compute/math_functor.hpp"
+#include "cell/warp.hpp"
+#include "cuda_utils.hpp"
+#include "types/layout.hpp"
+
+namespace tiledcuda::cell::compute {
+
+namespace tl = tile_layout;
+
+namespace detail {
+
+template <typename RegTile, typename Functor>
+struct ElementWise {
+    using DType = typename RegTile::DType;
+
+    static constexpr int kRows = RegTile::kRows;
+    static constexpr int kCols = RegTile::kCols;
+
+    DEVICE void operator()(const RegTile& src, RegTile& dst) {
+        Functor f;
+#pragma unroll
+        for (int i = 0; i < kRows; ++i) {
+#pragma unroll
+            for (int j = 0; j < kCols; ++j) {
+                // dst(i, j) = f(src(i, j));
```

Comment:
Is this commented line still necessary, if not, it canbe deleted.

Reply:
Deleted.